tests/behaviour/steps/connection/transaction.rs

```diff
@@ -18,6 +18,7 @@ use crate::{
 #[apply(generic_step)]
 #[step(expr = "connection open {word} transaction for database: {word}")]
 pub async fn connection_open_transaction(context: &mut Context, tx_type: String, db_name: String) {
+    assert!(context.transaction().is_none(), "Have to close previous transaction");
```

Comment:
I prefer the assertion/error message style like "Previous/Existing transaction must be closed" but not any kind of issue really

Reply:
Gp, fixed